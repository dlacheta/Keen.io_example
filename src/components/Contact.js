import React from 'react';

const Contact = () => (
  <React.Fragment>
    <h2>Contact</h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vel varius nulla, nec
      vestibulum tortor. Ut erat velit, eleifend sit amet tristique id, eleifend ut turpis.
      Phasellus nec velit id sapien viverra faucibus. Lorem ipsum dolor sit amet, consectetur
      adipiscing elit. In luctus sem at erat viverra, ut aliquet quam sollicitudin. Cras dictum ex a
      metus pellentesque accumsan. Cras euismod facilisis nunc, in eleifend augue. Nunc rhoncus nunc
      vel massa dictum, id commodo magna commodo. Maecenas eleifend enim interdum odio porta,
      aliquam convallis nunc vulputate. Nulla facilisi. In hac habitasse platea dictumst. Etiam
      pharetra mauris ex, eget consequat nulla aliquam vel.
    </p>
    <p>
      Mauris a sollicitudin magna. Integer dignissim gravida eleifend. Pellentesque vel justo diam.
      Nulla semper eget mi nec pellentesque. Donec in nisi at lacus elementum viverra sit amet eget
      elit. Aliquam nisi orci, fringilla ut luctus nec, tempus sit amet quam. Curabitur eu
      vestibulum nisi, vel dictum augue.
    </p>
  </React.Fragment>
);

export default Contact;
