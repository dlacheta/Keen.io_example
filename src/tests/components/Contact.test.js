import React from 'react';
import { shallow } from 'enzyme';
import Contact from '../../components/Contact';

it('Contact component renders correctly', () => {
  const wrapper = shallow(<Contact />);
  expect(wrapper).toMatchSnapshot();
});
