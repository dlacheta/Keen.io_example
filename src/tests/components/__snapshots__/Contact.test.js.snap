// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Contact />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h2>
          Contact
        </h2>,
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vel varius nulla, nec vestibulum tortor. Ut erat velit, eleifend sit amet tristique id, eleifend ut turpis. Phasellus nec velit id sapien viverra faucibus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In luctus sem at erat viverra, ut aliquet quam sollicitudin. Cras dictum ex a metus pellentesque accumsan. Cras euismod facilisis nunc, in eleifend augue. Nunc rhoncus nunc vel massa dictum, id commodo magna commodo. Maecenas eleifend enim interdum odio porta, aliquam convallis nunc vulputate. Nulla facilisi. In hac habitasse platea dictumst. Etiam pharetra mauris ex, eget consequat nulla aliquam vel.
        </p>,
        <p>
          Mauris a sollicitudin magna. Integer dignissim gravida eleifend. Pellentesque vel justo diam. Nulla semper eget mi nec pellentesque. Donec in nisi at lacus elementum viverra sit amet eget elit. Aliquam nisi orci, fringilla ut luctus nec, tempus sit amet quam. Curabitur eu vestibulum nisi, vel dictum augue.
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Contact",
        },
        "ref": null,
        "rendered": "Contact",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vel varius nulla, nec vestibulum tortor. Ut erat velit, eleifend sit amet tristique id, eleifend ut turpis. Phasellus nec velit id sapien viverra faucibus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In luctus sem at erat viverra, ut aliquet quam sollicitudin. Cras dictum ex a metus pellentesque accumsan. Cras euismod facilisis nunc, in eleifend augue. Nunc rhoncus nunc vel massa dictum, id commodo magna commodo. Maecenas eleifend enim interdum odio porta, aliquam convallis nunc vulputate. Nulla facilisi. In hac habitasse platea dictumst. Etiam pharetra mauris ex, eget consequat nulla aliquam vel.",
        },
        "ref": null,
        "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vel varius nulla, nec vestibulum tortor. Ut erat velit, eleifend sit amet tristique id, eleifend ut turpis. Phasellus nec velit id sapien viverra faucibus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In luctus sem at erat viverra, ut aliquet quam sollicitudin. Cras dictum ex a metus pellentesque accumsan. Cras euismod facilisis nunc, in eleifend augue. Nunc rhoncus nunc vel massa dictum, id commodo magna commodo. Maecenas eleifend enim interdum odio porta, aliquam convallis nunc vulputate. Nulla facilisi. In hac habitasse platea dictumst. Etiam pharetra mauris ex, eget consequat nulla aliquam vel.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Mauris a sollicitudin magna. Integer dignissim gravida eleifend. Pellentesque vel justo diam. Nulla semper eget mi nec pellentesque. Donec in nisi at lacus elementum viverra sit amet eget elit. Aliquam nisi orci, fringilla ut luctus nec, tempus sit amet quam. Curabitur eu vestibulum nisi, vel dictum augue.",
        },
        "ref": null,
        "rendered": "Mauris a sollicitudin magna. Integer dignissim gravida eleifend. Pellentesque vel justo diam. Nulla semper eget mi nec pellentesque. Donec in nisi at lacus elementum viverra sit amet eget elit. Aliquam nisi orci, fringilla ut luctus nec, tempus sit amet quam. Curabitur eu vestibulum nisi, vel dictum augue.",
        "type": "p",
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h2>
            Contact
          </h2>,
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vel varius nulla, nec vestibulum tortor. Ut erat velit, eleifend sit amet tristique id, eleifend ut turpis. Phasellus nec velit id sapien viverra faucibus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In luctus sem at erat viverra, ut aliquet quam sollicitudin. Cras dictum ex a metus pellentesque accumsan. Cras euismod facilisis nunc, in eleifend augue. Nunc rhoncus nunc vel massa dictum, id commodo magna commodo. Maecenas eleifend enim interdum odio porta, aliquam convallis nunc vulputate. Nulla facilisi. In hac habitasse platea dictumst. Etiam pharetra mauris ex, eget consequat nulla aliquam vel.
          </p>,
          <p>
            Mauris a sollicitudin magna. Integer dignissim gravida eleifend. Pellentesque vel justo diam. Nulla semper eget mi nec pellentesque. Donec in nisi at lacus elementum viverra sit amet eget elit. Aliquam nisi orci, fringilla ut luctus nec, tempus sit amet quam. Curabitur eu vestibulum nisi, vel dictum augue.
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Contact",
          },
          "ref": null,
          "rendered": "Contact",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vel varius nulla, nec vestibulum tortor. Ut erat velit, eleifend sit amet tristique id, eleifend ut turpis. Phasellus nec velit id sapien viverra faucibus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In luctus sem at erat viverra, ut aliquet quam sollicitudin. Cras dictum ex a metus pellentesque accumsan. Cras euismod facilisis nunc, in eleifend augue. Nunc rhoncus nunc vel massa dictum, id commodo magna commodo. Maecenas eleifend enim interdum odio porta, aliquam convallis nunc vulputate. Nulla facilisi. In hac habitasse platea dictumst. Etiam pharetra mauris ex, eget consequat nulla aliquam vel.",
          },
          "ref": null,
          "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vel varius nulla, nec vestibulum tortor. Ut erat velit, eleifend sit amet tristique id, eleifend ut turpis. Phasellus nec velit id sapien viverra faucibus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In luctus sem at erat viverra, ut aliquet quam sollicitudin. Cras dictum ex a metus pellentesque accumsan. Cras euismod facilisis nunc, in eleifend augue. Nunc rhoncus nunc vel massa dictum, id commodo magna commodo. Maecenas eleifend enim interdum odio porta, aliquam convallis nunc vulputate. Nulla facilisi. In hac habitasse platea dictumst. Etiam pharetra mauris ex, eget consequat nulla aliquam vel.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Mauris a sollicitudin magna. Integer dignissim gravida eleifend. Pellentesque vel justo diam. Nulla semper eget mi nec pellentesque. Donec in nisi at lacus elementum viverra sit amet eget elit. Aliquam nisi orci, fringilla ut luctus nec, tempus sit amet quam. Curabitur eu vestibulum nisi, vel dictum augue.",
          },
          "ref": null,
          "rendered": "Mauris a sollicitudin magna. Integer dignissim gravida eleifend. Pellentesque vel justo diam. Nulla semper eget mi nec pellentesque. Donec in nisi at lacus elementum viverra sit amet eget elit. Aliquam nisi orci, fringilla ut luctus nec, tempus sit amet quam. Curabitur eu vestibulum nisi, vel dictum augue.",
          "type": "p",
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
